src/core/emulator.cpp

```diff
@@ -1624,6 +1624,16 @@ GraphicsSynthesizer& Emulator::get_gs()
     return gs;
 }
 
+EEBreakpointList* Emulator::get_ee_breakpoint_list()
+{
+    return &ee_breakpoints;
+}
+
+IOPBreakpointList* Emulator::get_iop_breakpoint_list()
```

Comment:
Recommend marking as `const`.

```suggestion
IOPBreakpointList* Emulator::get_iop_breakpoint_list() const
```

Reply:
same as `ee_breakpoint_list`